src/http2_frame_window_update.erl

```diff
@@ -30,10 +30,13 @@ read_binary(_, _) ->
 send({Transport, Socket}, #window_update{window_size_increment=Payload}, StreamId) ->
     Transport:send(Socket, [
         <<4:24,?WINDOW_UPDATE:8,0:8,0:1,StreamId:31>>,
-        <<0:1,Payload:31>>]).
+                            <<0:1,Payload:31>>]);
+send({Transport, Socket}, Size, StreamId) ->
```

Comment:
maybe this should be a different function, since the second argument is a very different type